Gemfile.lock

```diff
@@ -1,78 +1,78 @@
 GEM
   remote: https://rubygems.org/
   specs:
-    actionpack (6.0.0)
-      actionview (= 6.0.0)
-      activesupport (= 6.0.0)
-      rack (~> 2.0)
+    actionpack (6.0.2.1)
+      actionview (= 6.0.2.1)
+      activesupport (= 6.0.2.1)
+      rack (~> 2.0, >= 2.0.8)
       rack-test (>= 0.6.3)
       rails-dom-testing (~> 2.0)
       rails-html-sanitizer (~> 1.0, >= 1.2.0)
-    actionview (6.0.0)
-      activesupport (= 6.0.0)
+    actionview (6.0.2.1)
+      activesupport (= 6.0.2.1)
       builder (~> 3.1)
       erubi (~> 1.4)
       rails-dom-testing (~> 2.0)
       rails-html-sanitizer (~> 1.1, >= 1.2.0)
-    activemodel (6.0.0)
-      activesupport (= 6.0.0)
-    activerecord (6.0.0)
-      activemodel (= 6.0.0)
-      activesupport (= 6.0.0)
-    activesupport (6.0.0)
+    activemodel (6.0.2.1)
+      activesupport (= 6.0.2.1)
+    activerecord (6.0.2.1)
+      activemodel (= 6.0.2.1)
+      activesupport (= 6.0.2.1)
+    activesupport (6.0.2.1)
       concurrent-ruby (~> 1.0, >= 1.0.2)
       i18n (>= 0.7, < 2)
       minitest (~> 5.1)
       tzinfo (~> 1.1)
-      zeitwerk (~> 2.1, >= 2.1.8)
-    builder (3.2.3)
+      zeitwerk (~> 2.2)
+    builder (3.2.4)
     concurrent-ruby (1.1.5)
-    crass (1.0.4)
+    crass (1.0.5)
     double_entry (1.0.1)
       activerecord (>= 3.2.0)
       activesupport (>= 3.2.0)
       money (>= 6.0.0)
       railties (>= 3.2.0)
-    erubi (1.8.0)
-    i18n (1.6.0)
+    erubi (1.9.0)
+    i18n (1.7.0)
       concurrent-ruby (~> 1.0)
-    json (2.2.0)
+    json (2.3.0)
     jwt (2.1.0)
     jwt_signed_request (2.5.1)
       jwt (>= 1.5.0, < 2.2.0)
       rack
-    loofah (2.2.3)
+    loofah (2.4.0)
       crass (~> 1.0.2)
       nokogiri (>= 1.5.9)
     method_source (0.9.2)
     mini_portile2 (2.4.0)
-    minitest (5.11.3)
-    money (6.13.4)
+    minitest (5.13.0)
+    money (6.13.6)
       i18n (>= 0.6.4, <= 2)
-    nokogiri (1.10.4)
+    nokogiri (1.10.7)
       mini_portile2 (~> 2.4.0)
     pagerduty (2.1.2)
       json (>= 1.7.7)
-    rack (2.0.7)
+    rack (2.0.8)
     rack-test (1.1.0)
       rack (>= 1.0, < 3)
     rails-dom-testing (2.0.3)
       activesupport (>= 4.2.0)
       nokogiri (>= 1.6)
-    rails-html-sanitizer (1.2.0)
-      loofah (~> 2.2, >= 2.2.2)
-    railties (6.0.0)
-      actionpack (= 6.0.0)
-      activesupport (= 6.0.0)
+    rails-html-sanitizer (1.3.0)
+      loofah (~> 2.3)
+    railties (6.0.2.1)
+      actionpack (= 6.0.2.1)
+      activesupport (= 6.0.2.1)
       method_source
       rake (>= 0.8.7)
       thor (>= 0.20.3, < 2.0)
-    rake (12.3.3)
-    thor (0.20.3)
+    rake (13.0.1)
```

Comment:
### [rake](https://github.com/ruby/rake)

**Major** version upgrade :chart_with_upwards_trend::exclamation: 12.3.3 → 13.0.1

[_[change-log](https://github.com/ruby/rake/blob/v13.0.1/History.rdoc), [source-code](https://github.com/ruby/rake/tree/v13.0.1)_]





<details>
<summary>Commits</summary>

A change of **53** commits. See the full changes on [the compare page](https://github.com/ruby/rake/compare/v12.3.3...v13.0.1).

These are the first 10 commits:
- (449766f) [Add order only dependency.](https://github.com/ruby/rake/commit/449766fb9abbfb9d8365ff824952cd1aa9067683)
- (714a180) [Fixed bug: Task raises previous exception on second invokati…](https://github.com/ruby/rake/commit/714a18093b38661508737a6849fc7168f28829a1)
- (282b0d3) [Applied requested changes of @yuki24: Chose clean git blame …](https://github.com/ruby/rake/commit/282b0d31586c7b723f6ba7d5103f874adec1bdb9)
- (382e804) [feat: add `without_parent_dir` to `PackageTask`](https://github.com/ruby/rake/commit/382e8047208675c9fb09e2cf9e5c4e4d3fe7ac5f)
- (a232f02) [Update ruby.yml](https://github.com/ruby/rake/commit/a232f0204c636dc6b42c7bffef93c1d858635a05)
- (cfc7e48) [Enabled build matrix.](https://github.com/ruby/rake/commit/cfc7e48a0447cec007c9e872035e846d7fd45ff5)
- (8017d98) [Added Windows and macOS.](https://github.com/ruby/rake/commit/8017d98af33f9bd7e626afe79c09eb1c97c8ec22)
- (4206043) [Fixed build names.](https://github.com/ruby/rake/commit/42060431d50fb50c6a2c9dd38fb68c3e5890671c)
- (a4dc9e0) [Windows env only provide Ruby 2.4+](https://github.com/ruby/rake/commit/a4dc9e07dc007937137779cf564aae657b4ed025)
- (0544f30) [setup-ruby is not support macOS env.](https://github.com/ruby/rake/commit/0544f30f32a4029f50c7f2a8233e8ce7b0ff71f8)


</details>